package.json

```
@@ -8,6 +8,7 @@
   "dependencies": {
     "@formatjs/intl-pluralrules": "1.3.9",
     "@formatjs/intl-relativetimeformat": "4.5.1",
+    "@types/classnames": "^2.2.6",
```

Looks like this is not needed.  Could you please confirm?

---

It is needed, we're using the `classNames` library in `center_channel.tsx`

---

Thanks @devinbinnie.  Then, my only request is to have it on fix version
```suggestion
    "@types/classnames": "2.2.6",
```

---

Agreed.